src/main/resources/public/ts/models/slot.model.ts

```diff
@@ -0,0 +1,33 @@
+export interface ISlotResponse {
+    id: String;
```

Comment:
Est ce que c'est voulu tout les typages "String" et pas "string" ?

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html